gallerypage.py

```
@@ -1144,36 +1042,73 @@ def _write_jekyll(self, filepath, remote_keys, replacements,
+            for key in remote_keys:
+                if 'movie' in key.lower():
+                    icon = '<img src="/icons-local/movie_icon.png" /> '
+                else:
+                    icon = ''
+
+                try:
+                    (url, shape, size) = self.remote_version_info[key]
+                except KeyError:
+                    continue
+
+                f.write(' * %s[%s]({{ site.baseurl }}%s){:target="_blank"} ' % (icon, key, url))
```

Ruff (0.15.2)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)